src/dnddiceroller/DiceTower.java

```diff
@@ -5,24 +6,79 @@
 
 /**
  * Dice Tower.
- * A Dice Tower is a tool used by serious gamers use to roll many dice at once.
- * It looks like this https://www.miniaturescenery.com/Images/PortableDiceTowerLarge.jpg
- * An instance of a dice tower is defined by the number panels it contains to help
- * provide a more regular distribution of die values. The die bounce from panel to
- * panel until they exit the dice tower at the bottom tray.
- * A dice tower will accept a collection of dice and reports their results when
- * they reach the tray at the bottom
- * @author Paul Scarrone
+ * A Dice Tower is a tool used by serious gamers to roll many dice at once.
+ *
+ * An instance of a dice tower is defined by 
+ *      the number of panels it contains to help provide a regular
+ *      distribution of die values.
+ * 
+ * The die bounce from panel to panel until they exit the dice tower
+ * at the bottom of they tray.
+ * 
+ * A dice tower will accept a collection of dice and report their results
+ * when they reach the tray at the bottom
  */
-public class DiceTower {
-  final int PANEL_COUNT = 3;
-  List<Die> dice;
 
-  public DiceTower() {
-	this.dice = new ArrayList();
-  }
-  
-  public DiceTower(List dice) {
-	this.dice = dice;
-  }
-}
+/**
+ *
+ * @author Rhonda Marshall
+ */
+public class DiceTower {
+    private final int PANEL_COUNT = 3;
+    List<Die> dice;
+    int trayValue;
+    private final Boolean debug;   // This is for my purposes to debug this progam
```

Comment:
So I like the debug flag for the class to help with tracking the output.
Here is some advice when using this kind of logic. Your constructors do make sure they manage the state of debug as disabled for normal operations of the class. :+1: 

What this flag really does is change the `verbosity` of the class. So in criticism of this should be that we would call this the `verbose` flag instead of the `debug` flag. That tells me what effect I am to expect from this flag.
My other criticism is this is only modifiable from the code right now.

The rule is if we don't use it we delete it before we close our Pull Request. Because I don't see a way to toggle the classes verbosity without recompiling the code I would say this should have been deleted.